core/inject.js

```diff
@@ -78,6 +82,14 @@ Blockly.inject = function(container, opt_options) {
 
   Blockly.svgResize(workspace);
 
+  subContainer.onfocus = function() {
```

Comment:
Use addEventListener instead (more generally, so others are able to add other listeners if they want).